change/@fluentui-react-context-selector-899c905b-9a1b-45e8-b5f6-6e4e260e9bad.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "feat: Overrides context (unstable)",
```

```suggestion
  "type": "none",
  "comment": "chore: fix a typo in jsdoc",
```

This change file is not correct as it's only about a typo